bitnami/fluentd/templates/aggregator-statefulset.yaml

```diff
@@ -0,0 +1,116 @@
+          ports:
+            - name: http
+              containerPort: 9880
+              protocol: TCP
+            - name: tcp
+              containerPort: {{ .Values.aggregator.port }}
+              protocol: TCP
+            {{- if .Values.metrics.enabled }}
+            - name: metrics
+              containerPort: {{ .Values.metrics.service.port }}
+              protocol: TCP
+            {{- end }}
+          {{- if .Values.aggregator.livenessProbe.enabled }}
+          livenessProbe:
+            httpGet:
+              path: /fluentd.healthcheck?json=%7B%22ping%22%3A+%22pong%22%7D
+              port: http
+            initialDelaySeconds: {{ .Values.aggregator.livenessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.aggregator.livenessProbe.periodSeconds }}
+            timeoutSeconds: {{ .Values.aggregator.livenessProbe.timeoutSeconds }}
+            successThreshold: {{ .Values.aggregator.livenessProbe.successThreshold }}
+            failureThreshold: {{ .Values.aggregator.livenessProbe.failureThreshold }}
+          {{- end }}
+          {{- if .Values.aggregator.readinessProbe.enabled }}
+          readinessProbe:
+            httpGet:
+              path: /fluentd.healthcheck?json=%7B%22ping%22%3A+%22pong%22%7D
+              port: http
+            initialDelaySeconds: {{ .Values.aggregator.readinessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.aggregator.readinessProbe.periodSeconds }}
+            timeoutSeconds: {{ .Values.aggregator.readinessProbe.timeoutSeconds }}
+            successThreshold: {{ .Values.aggregator.readinessProbe.successThreshold }}
+            failureThreshold: {{ .Values.aggregator.readinessProbe.failureThreshold }}
+          {{- end }}
+          resources: {{ toYaml .Values.aggregator.resources | nindent 12 }}
+          volumeMounts:
+            - name: fluentd-config
+              mountPath: /opt/bitnami/fluentd/conf
+            - name: buffer
+              mountPath: /opt/bitnami/fluentd/logs/buffers
+            {{- if .Values.tls.enabled }}
+            - name: {{ template "fluentd.fullname" . }}-certs
+              mountPath: /opt/bitnami/fluentd/certs
+            {{- end }}
+      volumes:
+        {{- if .Values.tls.enabled }}
+        - name: {{ template "fluentd.fullname" . }}-certs
+          secret:
+            secretName: {{ if .Values.tls.existingSecret }}{{ .Values.tls.existingSecret }}{{- else }}{{ template "fluentd.fullname" . }}-certs{{- end }}
+            items:
+            - key: ca_certificate.pem
+              path: ca_certificate.pem
+            - key: server_certificate.pem
+              path: server_certificate.pem
+            - key: server_key.pem
+              path: server_key.pem
+        {{- end }}
+        - name: fluentd-config
+          configMap:
+            name: {{ if .Values.aggregator.configMap }}{{ .Values.aggregator.configMap }}{{- else }}{{ include "fluentd.fullname" . }}-aggregator-cm{{- end }}
```

Comment:
Move this to a helper. This could be very useful for umbrella charts, for example. Check PostgreSQL as an example

Reply:
```
name: {{ include "fluentd.aggregatorCM" . }}
```
For example